vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt

```diff
@@ -236,7 +236,7 @@ class CreateRoomViewModel @AssistedInject constructor(@Assisted val initialState
                         if (initialState.parentSpaceId != null) {
                             // add it as a child
                             try {
-                                val via = session.sessionParams.homeServerHost?.let { listOf(it) } ?: emptyList()
+                                val via = session.sessionParams.homeServerHost?.let { listOf(it) }.orEmpty()
```

Comment:
there was another change in my review: use `toList()`. Is it possible?

Reply:
:D

Reply:
Oh my bad, this method really sucks for Strings

Reply:
maybe create an extension `fun String.asList() = listOf(this)` :)

Reply:
TBH, i think i will refactor this soon and see how to reuse the permalink service computeViaParams